cmd/config/docs/api-conventions/config-fn.md

```diff
@@ -222,15 +222,15 @@ Following is an example of running the `kustomize config run` using the precedin
     metadata:
       name: my-instance
       annotations:
+        config.k8s.io/function: |
```

Comment:
can we make the annotation names consistent -- e.g. either both kubernetes.io or both k8s.io?

Reply:
Yes k8s.io is widely adopted so we should change config.kubernetes.io/local-config to config.k8s.io/local-config. Renaming involves a few dozen files so it's out of the scope of this PR. I made a separate issue for this https://github.com/kubernetes-sigs/kustomize/issues/2033